samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,51 +1,53 @@
-import { expect, test } from '@playwright/test';
+/* eslint-disable testing-library/prefer-screen-queries */
+import { test } from '@playwright/test';
+import * as util from './utils'
+import * as simpletests from './testsBasic'
+import * as plannertests from './testsPlanner'
+import * as mutests from './testsMultiuser'
 
-/*
-This test is a simple end-to-end test that verifies the app can be
-loaded, and signed in. The user can send a message to the bot, and expect
-a response.
-*/
-test('get response from bot', async ({ page }) => {
-    // Make sure the server is running.
-    await page.goto('https://localhost:40443/healthz');
-    expect(page.getByText('Healthy')).toBeDefined();
+test.describe('Copilot Chat App Test Suite', () => {
+    // Note: A new chat session is opened for each test so that 
+    // the chat history is not polluted and the LLM is not confused.
 
-    await page.goto('/');
-    // Expect the page to contain a "Login" button.
-    await page.getByRole('button').click();
-    // Clicking the login button should redirect to the login page.
-    await expect(page).toHaveURL(new RegExp('^' + process.env.REACT_APP_AAD_AUTHORITY));
-    // Login with the test user.
-    await page.getByPlaceholder('Email, phone, or Skype').click();
-    await page.getByPlaceholder('Email, phone, or Skype').fill(process.env.REACT_APP_TEST_USER_ACCOUNT as string);
-    await page.getByRole('button', { name: 'Next' }).click();
-    await page.getByPlaceholder('Password').click();
-    await page.getByPlaceholder('Password').fill(process.env.REACT_APP_TEST_USER_PASSWORD as string);
-    await page.getByRole('button', { name: 'Sign in' }).click();
+    test.describe('A, runs in parallel with B', () => {
```

Comment:
in my comment on splitting up the files I assumed we'd break the `describe()` blocks too. for example:
```suggestion
    // basic.test.ts

    test.describe('Basic tests', () => {
        test('Basic Bot Responses, ...)
        // ... other basic tests
    });
```

and then the same structure for the other files.

Reply:
Why would you not want to parallelize tests?
This is meant to run after each pull request is merged into main (cant do it prior because it relies on secrets). We want the iteration time on these things to be as fast as possible.